airtable/backup_airtable.py

```diff
@@ -2,44 +2,18 @@
 import datetime
 import os
 import tempfile
-import urllib
 import zipfile
 
 from boto.s3.connection import S3Connection
 from boto.s3.key import Key
-import requests
+import airtable
```

Comment:
Nope! I grepped for the things I took out, but I didn't know what would happen if I ran it, so I didn't.